Content.Client/StatusIcon/StatusIconOverlay.cs

```diff
@@ -72,8 +74,16 @@ protected override void Draw(in OverlayDrawArgs args)
 
             foreach (var proto in icons)
             {
+
                 var texture = _sprite.Frame0(proto.Icon);
 
+                if (proto.Animated)
+                {
+                    var curTime = _timing.RealTime;
+                    texture = _sprite.GetFrame(proto.Icon, curTime);
+                }
+
+
```

Comment:
Just use getframe instead of getting the texture twice instead and use an RSI so it natively supports it rather than requiring additional fields.

Reply:
That is what I did originally, however I do not know how to set it to use an RSI. Seemingly changing the path for the icons in the yaml to /Textures/Interface/... instead of Interface/.... accomplished this, but I'm not sure why. Is there a way to accomplish that in code, without having to change every yaml file independently?